kong/plugins/aws-lambda/schema.lua

```diff
@@ -75,6 +75,7 @@ return {
           type = "boolean",
           default = false,
         } },
+        -- TODO: remove proxy_scheme in Kong 3.0
```

Comment:
@bungle We should explore a "deprecated = true" property on the schema to manage these programmatically.